mysql-schema/complaint-table.txt

```diff
@@ -3,12 +3,14 @@ CREATE DATABASE `hclecomm` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb
 
 
 
-CREATE TABLE `hclecomm`.`complaints` (
-  `id` INT NOT NULL AUTO_INCREMENT,
-  `first_name` VARCHAR(45) NULL,
-  `last_name` VARCHAR(45) NULL,
-  `email` VARCHAR(45) NULL,
-  `subject` VARCHAR(45) NULL,
-  `complaint` VARCHAR(500) NULL,
-  `status` VARCHAR(45) NULL,
-  PRIMARY KEY (`id`));
\ No newline at end of file
+CREATE TABLE `complaints` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(45) DEFAULT NULL,
+  `last_name` varchar(45) DEFAULT NULL,
+  `email` varchar(45) DEFAULT NULL,
+  `subject` varchar(45) DEFAULT NULL,
+  `complaint` varchar(500) DEFAULT NULL,
+  `status` varchar(45) DEFAULT NULL,
+  `closing_comment` varchar(500) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=24 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
```

Comment:
@rakeshkumar-verma : why do we need below statement in query : ENGINE=InnoDB AUTO_INCREMENT=24 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;

Reply:
@rishabh-behl  I have export the table schema so its exported db engine, auto increment count.... Removed now.